DNA_RNA_classes_first_commit.py

```diff
@@ -0,0 +1,44 @@
+class Dna(str):
+    
+    
+    def __init__(self, dna_seq):  
+        self.dna_seq = dna_seq

```

Comment:
If you applied "upper" string method here in __init__ part, you would escape repeating it in methods. Also, your list of codes will contain only uppercase letters.